### PR TITLE
Adds beauty and the beast

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
I legitimately hate this thing.
I still added it in it's full functionality.

New players will cry. 
They can email all complaints and tears to contact@projmoon.com

## Why It's Good For The Game
It's not. But another abnormality is.

## Changelog
:cl:
add: Added Beauty and the beast
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	maxHealth = 650
+	health = 650
+	threat_level = TETH_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(40, 20, -20, -20, -20),
+		ABNORMALITY_WORK_INSIGHT = list(50, 50, 40, 30, 30),
+		ABNORMALITY_WORK_ATTACHMENT = list(30, 15, -50, -50, -50),
+		ABNORMALITY_WORK_REPRESSION = list(65, 65, 65, 65, 65)
+		)
+	work_damage_amount = 3
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm:15]: The damage is rather low, isn't it?

Comment (Kitsunemitsu) [reply code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm:15]: Right, sorry i just plugged in the original value here

File: code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			//If you already did repression, die. Fuck you.
+			user.gib()
+			dead = TRUE
+			injured = FALSE
+
+			icon_state = "beauty_dead"
+			animate(src, alpha = 0, time = 10 SECONDS)
+			QDEL_IN(src, 10 SECONDS)
+
+			dead = FALSE
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm:42]: Why is it here? QDEL_IN is a timer, so you are effectively nullifying what you've done above.

File: code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+	else
+		injured = FALSE
+		icon_state = "beauty"
+	return
+
+
+/mob/living/simple_animal/hostile/abnormality/beauty/proc/attempt_work(mob/living/carbon/human/user, work_type)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/teth/beauty_beast.dm:50]:
```suggestion
/mob/living/simple_animal/hostile/abnormality/beauty/attempt_work(mob/living/carbon/human/user, work_type)
```
No need to add "/proc/" if it has a parent proc defined already.

**Pull request merged**